### ISSUE TITLE
Import Error for MediaBlackhole

### ISSUE THREAD

**Issue by sarkarchandan** [jlaine/aiortc#79]
Hi Jeremy,

I was trying the new aiortc 0.9.9 video streams example. It is throwing error while importing the MediaBlackhole from media.py. I have installed aiortc in venv and can see the existing class MediaBlackhole in the media.py inside contrib.

`from aiortc.contrib.media import MediaBlackhole, MediaRecorder`

Traceback (most recent call last):
  File "stream.py", line 7, in <module>
    from aiortc.contrib.media import MediaBlackhole, MediaRecorderImportError: cannot import name 'MediaBlackhole' from 'aiortc.contrib.media' (/usr/local/lib/python3.7/site-packages/aiortc/contrib/media.py)

Could you please suggest how to troubleshoot this ?

Thank you.

**Comment by jlaine** [comment]
Are you sure you've actually got aiortc version 0.9.9 installed?

Works for me in a fresh virtualenv:

```
sharky@mango:/tmp$ python3 -m venv aiortc-test
(aiortc-test) sharky@mango:/tmp$ pip install aiortc
[...]
(aiortc-test) sharky@mango:/tmp$ python
Python 3.6.6 (default, Jun 27 2018, 14:44:17) 
[GCC 8.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from aiortc.contrib.media import MediaBlackhole
>>> 
```

**Comment by sarkarchandan** [comment]
I executed this command which automatically grabbed aiortc-0.9.9.tar.gz
./venv/bin/python -m pip install -U aiortc
Collecting aiortc
  Downloading https://files.pythonhosted.org/packages/f1/57/99a3fd2c3e1c29e732bd206b60492416ae424d664c9c5b1ddf12d5ce8060/aiortc-0.9.9.tar.gz (1.1MB)
    100% |████████████████████████████████| 1.1MB 7.6MB/s
Collecting aioice<0.7.0,>=0.6.7 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/34/91/4e6bcfedd65c4b992b79d593f226b2434146f0fc3a8ad0205eedfa8ce011/aioice-0.6.8-py3-none-any.whl
Collecting attrs (from aiortc)
  Using cached https://files.pythonhosted.org/packages/3a/e1/5f9023cc983f1a628a8c2fd051ad19e76ff7b142a0faf329336f9a62a514/attrs-18.2.0-py2.py3-none-any.whl
Collecting av<0.6.0,>=0.5.3 (from aiortc)
Collecting cffi>=1.0.0 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/0b/ba/32835c9965d8a0090723e1d0b47373365525c4bd08c807b5efdc9fecbc99/cffi-1.11.5-cp37-cp37m-macosx_10_9_x86_64.whl
Collecting crcmod (from aiortc)
Collecting cryptography>=2.2 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/98/0b/a6f293e5f10095dd8657a1b125c1ba6995c59d39cd8e20355475c8f760d0/cryptography-2.3.1-cp34-abi3-macosx_10_6_intel.whl
Collecting numpy (from aiortc)
  Using cached https://files.pythonhosted.org/packages/f9/64/f47c172c2d2ee8907b5918ff7af7e52207f6bf4a57983e4474fcda728112/numpy-1.15.2-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting pyee (from aiortc)
  Using cached https://files.pythonhosted.org/packages/8e/06/10c18578e2d8b9cf9902f424f86d433c647ca55e82293100f53e6c0afab4/pyee-5.0.0-py2.py3-none-any.whl
Collecting pylibsrtp>=0.5.6 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/d2/66/ac460babd07ae3f727c3cb83c2fbc27f7f6f312360883d08d1fa9dd05f81/pylibsrtp-0.6.0-cp37-cp37m-macosx_10_6_intel.whl
Collecting pyopenssl (from aiortc)
  Using cached https://files.pythonhosted.org/packages/96/af/9d29e6bd40823061aea2e0574ccb2fcf72bfd6130ce53d32773ec375458c/pyOpenSSL-18.0.0-py2.py3-none-any.whl
Collecting netifaces (from aioice<0.7.0,>=0.6.7->aiortc)
Collecting pycparser (from cffi>=1.0.0->aiortc)
  Downloading https://files.pythonhosted.org/packages/68/9e/49196946aee219aead1290e00d1e7fdeab8567783e83e1b9ab5585e6206a/pycparser-2.19.tar.gz (158kB)
    100% |████████████████████████████████| 163kB 975kB/s
Collecting six>=1.4.1 (from cryptography>=2.2->aiortc)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting idna>=2.1 (from cryptography>=2.2->aiortc)

**Comment by sarkarchandan** [comment]
This is from the aiortc-0.9.9-py3.7.egg-info/PKG-INFO

Metadata-Version: 1.1
Name: aiortc
Version: 0.9.9
Summary: An implementation of WebRTC and ORTC
Home-page: https://github.com/jlaine/aiortc
Author: Jeremy Lainé
Author-email: jeremy.laine@m4x.org
License: BSD

**Comment by jlaine** [comment]
Ah wait, could you paste a bit more of the traceback? It looks your issue might be with PyAV, not aiortc itself.

**Comment by sarkarchandan** [comment]
My av version is 0.5.3

**Comment by sarkarchandan** [comment]
I guess I pasted the whole traceback that I got by executing the pip install.

**Comment by sarkarchandan** [comment]
I did that again freshly now

./venv/bin/python -m pip install -U aiortc 
Collecting aiortc
  Using cached https://files.pythonhosted.org/packages/f1/57/99a3fd2c3e1c29e732bd206b60492416ae424d664c9c5b1ddf12d5ce8060/aiortc-0.9.9.tar.gz
Collecting aioice<0.7.0,>=0.6.7 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/34/91/4e6bcfedd65c4b992b79d593f226b2434146f0fc3a8ad0205eedfa8ce011/aioice-0.6.8-py3-none-any.whl
Collecting attrs (from aiortc)
  Using cached https://files.pythonhosted.org/packages/3a/e1/5f9023cc983f1a628a8c2fd051ad19e76ff7b142a0faf329336f9a62a514/attrs-18.2.0-py2.py3-none-any.whl
Collecting av<0.6.0,>=0.5.3 (from aiortc)
Collecting cffi>=1.0.0 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/0b/ba/32835c9965d8a0090723e1d0b47373365525c4bd08c807b5efdc9fecbc99/cffi-1.11.5-cp37-cp37m-macosx_10_9_x86_64.whl
Collecting crcmod (from aiortc)
Collecting cryptography>=2.2 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/98/0b/a6f293e5f10095dd8657a1b125c1ba6995c59d39cd8e20355475c8f760d0/cryptography-2.3.1-cp34-abi3-macosx_10_6_intel.whl
Collecting numpy (from aiortc)
  Using cached https://files.pythonhosted.org/packages/f9/64/f47c172c2d2ee8907b5918ff7af7e52207f6bf4a57983e4474fcda728112/numpy-1.15.2-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting pyee (from aiortc)
  Using cached https://files.pythonhosted.org/packages/8e/06/10c18578e2d8b9cf9902f424f86d433c647ca55e82293100f53e6c0afab4/pyee-5.0.0-py2.py3-none-any.whl
Collecting pylibsrtp>=0.5.6 (from aiortc)
  Using cached https://files.pythonhosted.org/packages/d2/66/ac460babd07ae3f727c3cb83c2fbc27f7f6f312360883d08d1fa9dd05f81/pylibsrtp-0.6.0-cp37-cp37m-macosx_10_6_intel.whl
Collecting pyopenssl (from aiortc)
  Using cached https://files.pythonhosted.org/packages/96/af/9d29e6bd40823061aea2e0574ccb2fcf72bfd6130ce53d32773ec375458c/pyOpenSSL-18.0.0-py2.py3-none-any.whl
Collecting netifaces (from aioice<0.7.0,>=0.6.7->aiortc)
Collecting pycparser (from cffi>=1.0.0->aiortc)
  Using cached https://files.pythonhosted.org/packages/68/9e/49196946aee219aead1290e00d1e7fdeab8567783e83e1b9ab5585e6206a/pycparser-2.19.tar.gz
Collecting six>=1.4.1 (from cryptography>=2.2->aiortc)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting idna>=2.1 (from cryptography>=2.2->aiortc)
  Using cached https://files.pythonhosted.org/packages/4b/2a/0276479a4b3caeb8a8c1af2f8e4355746a97fab05a372e4a2c6a6b876165/idna-2.7-py2.py3-none-any.whl
Collecting asn1crypto>=0.21.0 (from cryptography>=2.2->aiortc)
  Using cached https://files.pythonhosted.org/packages/ea/cd/35485615f45f30a510576f1a56d1e0a7ad7bd8ab5ed7cdc600ef7cd06222/asn1crypto-0.24.0-py2.py3-none-any.whl
Installing collected packages: netifaces, aioice, attrs, av, pycparser, cffi, crcmod, six, idna, asn1crypto, cryptography, numpy, pyee, pylibsrtp, pyopenssl, aiortc
  Running setup.py install for pycparser ... done
  Running setup.py install for aiortc ... done
Successfully installed aioice-0.6.8 aiortc-0.9.9 asn1crypto-0.24.0 attrs-18.2.0 av-0.5.3 cffi-1.11.5 crcmod-1.7 cryptography-2.3.1 idna-2.7 netifaces-0.10.7 numpy-1.15.2 pycparser-2.19 pyee-5.0.0 pylibsrtp-0.6.0 pyopenssl-18.0.0 six-1.11.0
You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

**Comment by sarkarchandan** [comment]
Hi Jeremy. I have figured out what was causing the problem. In my native installation I am still having the older version of aiortc. Since I have installed the newer version in virtual environment I needed to execute something like ./venv/bin/python cli.py instead of simply python cli.py which is still referring to the older version.
Sorry for confusing you. I am new in python, took some time to figure out the problem.
